fdbservice/FDBService.cpp

```diff
@@ -643,37 +643,33 @@ class FDBService : public CServiceBase {
 		return true;
 	}
 
-	const char* getValueMultiUtil(const CSimpleIni& ini, const char* name, ...) {
+	const char* getValueMulti(const CSimpleIni& ini, const char* name, ...) {
 		const char* ret = nullptr;
 		const char* section = nullptr;
 		va_list ap;
 		va_start(ap, name);
 		while (!ret && (section = va_arg(ap, const char*))) {
 			ret = ini.GetValue(section, name, nullptr);
 		}
-		va_end(ap);
-		return ret;
-	}
-
-	const char* getValueMulti(const CSimpleIni& ini, const char* name, ...) {
-		va_list args;
-		va_start(args, name);
-		const char* ret = nullptr;
-		ret = getValueMultiUtil(ini, name, args);
 		if (!ret) {
-			std::string nameCopy(name);
-			for (int i = nameCopy.size() - 1; i >= 0; --i) {
-				if (nameCopy[i] == '-') {
-					nameCopy.at(i) = '_';
+			std::string keyCopy(key);
```

Comment:
It doesn't look like there's anything named key here.

Reply:
Oh this is the old version of code, and I didn't fix bugs at that time. Please check the current update

Reply:
Sorry again!